src/reducers/rootReducer.js

```diff
@@ -1,8 +1,8 @@
-import { combineReducers } from 'redux'
-import weather from './weather'
+import { combineReducers } from 'redux';
+import localWeather from './local-weather';
 
 const rootReducer = combineReducers({
```

Comment:
You don't end up using this anymore—even in `index.js`. It's probably time for this to go.